frontend/package.json

```diff
@@ -9,11 +9,11 @@
     "@react-spring/web": "9.4.5",
```

Comment:
I am salty that we still have vulnerabilities in our repo (i think from ant-design/charts)

@gtangelo is it worth fixing?

Reply:
Hmm right now it shouldn't be a big issue right? If it worst down to it, we could probably remove `ant-design/charts` since there's only one component that relies on it as of this moment.